README.md

```diff
@@ -1,7 +1,13 @@
+<!-- Badges -->
+
+[![Build Status](https://github.com/ornlneutronimaging/bm3dornl/actions/workflows/actions.yml/badge.svg?branch=next)](https://github.com/ornlneutronimaging/bm3dornl/actions/workflows/actions.yml?query=branch?next)
```

Comment:
our workflow is not `actions.yml`, please use `unittest.yml` for the ci.cd badge

Reply:
Thanks, fixed